admin/constants.php

```diff
@@ -6,7 +6,11 @@
 
 <div id="hmbkp-constants">
 
-	<p><?php printf( __( 'You can %1$s any of the following %2$s in your %3$s to control advanced settings. %4$s. Defined %5$s will be highlighted.', 'backupwordpress' ), '<code>define</code>', '<code>' . __( 'Constants', 'backupwordpress' ) . '</code>', '<code>wp-config.php</code>', '<a href="http://codex.wordpress.org/Editing_wp-config.php">' . __( 'The Codex can help', 'backupwordpress' ) . '</a>', '<code>' . __( 'Constants', 'backupwordpress' ) . '</code>' ); ?></p>
+	<p><?php printf(
+		__( 'You can define any of the following constants in your %1$s file to control advanced settings. <a href="%2$s">The Codex can help</a>. Defined constants will be highlighted.', 'backupwordpress' ),
+		'<code>wp-config.php</code>',
+		'https://codex.wordpress.org/Editing_wp-config.php'
+	); ?></p>
```

Comment:
Escaping translations is something that I personally recommend strongly, but isn't strictly necessary. I think BWP should escape all of its translations, but that's something that would be better one all at once, and separately from these other enhancements.